scripts/git_log_to_transcripts.py

```diff
@@ -140,19 +140,56 @@ async def translate_commits_to_transcripts(repo, count=10):
                 "commit": sha,
                 "args": {},
                 "prompt": prompt,
-                "expected_edits": shown,
-                "expected_features": [
-                    str(f.relative_to(os.getcwd()))
-                    for f in bound_files(parsedLLMResponse.file_edits, padding=0)
-                ],
+                "expected_edits": llmResponse,
+            }
+            edited_files = {
+                str(f.relative_to(git_root))
+                for f in bound_files(parsedLLMResponse.file_edits, padding=0)
             }
+
+            # The longest context that could have been included to generate expected_edits
+            model = config.model
+            mentat_prompt_tokens = count_tokens(parser.get_system_prompt(), model)
+            expected_edits_tokens = count_tokens(benchmark["expected_edits"], model)
+            max_context_tokens = (
+                model_context_size(model) - mentat_prompt_tokens - expected_edits_tokens
+            )
+            # Fill-in available context
+            config.auto_tokens = 1e6
+            config.use_embeddings = True
+            code_context.use_llm = True
+            await code_context.get_code_message(
+                prompt, model, max_context_tokens, benchmark["expected_edits"]
+            )
+
+            git_root_length = len(str(git_root)) + 1
+            selected_features = [
+                f.ref()[git_root_length:] for f in code_context.features
+            ]
+            selected_files = {f.split(":")[0] for f in selected_features}
```

Comment:
When I run the benchmark I get errors like: `E       FileNotFoundError: [Errno 2] No such file or directory: '/home/jake/Development/mentat/tests/benchmarks/repos/for_transcripts/mentat/prompts/replacement_parser_prompt.txt'`. I think we need to filter out files that don't exist which happens when files are created as part of a PR. Sorry I probably should have done that in the original script.